pkg/valueset/postgresql.go

```diff
@@ -0,0 +1,33 @@
+package pkg
+
+import "github.com/cloudprivacylabs/lsa/layers/cmd/cmdutil"
+
+type config struct {
+	valuesets []valueset `yaml:"valuesets"`
+}
+
+type valueset struct {
```

Comment:
Define a valid exported struct for the configs, and unmarshal directly from yaml, add tags